src/NovelRunner_C.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by miron on 2/20/19.
+//
+
+#include "NovelRunner.h"
+#include "NovelRunner_C.h"
```

Comment:
Any chance we could change the name of this file to something that doesn't include an underscore here? make-do snake case is a little weird.